lib/src/utils.dart

```diff
@@ -3,9 +3,10 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:io';
 import 'dart:collection';
 
+import 'dart:io';
```

Comment:
```suggestion
import 'dart:io';
```

Reply:
Done! (pubspec was already set to an unreleased dev version, so I just added a changelog entry to the same one).

I fixed this newline locally to because last time I accepted a suggestion in a PR like this the CLA bot seems to get angry!